src/roiextractors/example_datasets/toy_example.py

```diff
@@ -7,10 +14,39 @@
 
 
 def _gaussian(x, mu, sigma):
+    """Compute classical gaussian with parameters x, mu, sigma."""
     return 1 / np.sqrt(2 * np.pi * sigma) * np.exp(-((x - mu) ** 2) / sigma)
 
 
 def _generate_rois(num_units=10, size_x=100, size_y=100, roi_size=4, min_dist=5, mode="uniform"):
+    """Generate ROIs with given parameters.
+
+    Parameters
+    ----------
+    num_units: int
+        Number of ROIs
+    size_x: int
+        Size of x dimension (pixels)
+    size_y: int
+        Size of y dimension (pixels)
+    roi_size: int
+        Siz of ROI in x and y dimension (pixels)
+    min_dist: int
+        Minimum distance between ROI centers (pixels)
+    mode: str
+        'uniform' or 'gaussian'.
```

Comment:
Stuff like this can - and in the Python 3.8 world - should become `typing.Literal`, e.g., `Literal["uniform", "gaussian"]`

I'm fine with this PR being the 'write things down as they are not as they should be', but that provides content for a next-step

Reply:
Agree. Added a TODO so we don't forget.